shared/campaigns.js

```diff
@@ -53,7 +53,13 @@ const CampaignMessageStatus = {
     COMPLAINED: 4,
     SCHEDULED: 5,
 
-    MAX: 5
+    MAX: 5,
```

Comment:
Please set MAX to 6

Reply:
Hi @bures ,

MAX wsa previously set to 5, what are the implications of changing it to 6? I'm not sure what the purpose of MAX is.

Reply:
MAX is used only to indicate the largest value of the enum. It's there if one needed to easily test that an option belongs to the enum.